action/protocol/execution/protocol_test.go

```diff
@@ -1018,6 +1049,86 @@ func TestIstanbulEVM(t *testing.T) {
 	})
 }
 
+func TestLondonEVM(t *testing.T) {
+	cfg := config.Default
+	config.SetEVMNetworkID(cfg.Chain.EVMNetworkID)
```

Comment:
`config.SetEVMNetworkID` is no longer needed after this commit: https://github.com/iotexproject/iotex-core/commit/7d6669869a568492206bc108dcdd3c0662f6f25f